packages/protocol-autonat/test/index.spec.ts

```diff
@@ -42,33 +39,30 @@ const defaultInit: AutoNATServiceInit = {
 
 describe('autonat', () => {
   let service: any
-  let components: Components
+  let components: AutoNATComponents
   let peerRouting: StubbedInstance<PeerRouting>
   let registrar: StubbedInstance<Registrar>
   let addressManager: StubbedInstance<AddressManager>
-  let connectionManager: StubbedInstance<DefaultConnectionManager>
+  let connectionManager: StubbedInstance<ConnectionManager>
   let transportManager: StubbedInstance<TransportManager>
-  let peerStore: StubbedInstance<PeerStore>
 
   beforeEach(async () => {
     peerRouting = stubInterface<PeerRouting>()
     registrar = stubInterface<Registrar>()
     addressManager = stubInterface<AddressManager>()
     addressManager.getAddresses.returns([])
 
-    connectionManager = stubInterface<DefaultConnectionManager>()
+    connectionManager = stubInterface<ConnectionManager>()
     transportManager = stubInterface<TransportManager>()
-    peerStore = stubInterface<PeerStore>()
 
-    components = defaultComponents({
+    components = {
       peerId: await createEd25519PeerId(),
       peerRouting,
       registrar,
       addressManager,
       connectionManager,
-      transportManager,
```

Comment:
the preference for no trailing commas makes this diff so bad...

Reply:
*cries in needless diff lines*

cc @achingbrain